lib/registrant.rb

```diff
@@ -0,0 +1,17 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data 
```

Comment:
Careful here, we only want to expose the methods that we need. We do not need a method called `permit` since we already have `permit?`